src/ducks/selection/SelectionBar.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useCallback } from 'react'
+
+import UISelectionBar from 'cozy-ui/transpiled/react/SelectionBar'
+import Alerter from 'cozy-ui/transpiled/react/Alerter'
+import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+
+import { useSelectionContext } from 'ducks/context/SelectionContext'
+import useSelectionBarActions from 'ducks/selection/useSelectionBarActions'
+import { useTransactionCategoryModal } from 'ducks/transactions/TransactionRow'
+
+const SelectionBar = () => {
+  const {
+    isSelectionModeEnabled,
+    isSelectionModeActive,
+    selected,
+    emptySelection
+  } = useSelectionContext()
+
+  const { t } = useI18n()
+
+  const beforeUpdate = useCallback(() => {
+    emptySelection()
+  }, [emptySelection])
```

Comment:
If emptySelection is not memoized in the context, the useCallback here will not work.